lesson24/src/js/main.js

```diff
@@ -73,4 +73,20 @@ submitButton.addEventListener("click", (e) => {
     window.location.href = "register-done.html";
 });
 
+const options = {
+    root: document.querySelector('[data-id="modal-inner"]'),
+};
```

Comment:
Q. Does this element exist?

Reply:
Thank you for your comment!
This element did not exist. This is a mistake and I  fixed.
https://github.com/nakagawamai/js-lesson/pull/26/commits/6fa7910d9e742ff97b8b86eeccf7475b66385c74